app/admin/tenants/onboarding.py

```diff
@@ -101,7 +108,11 @@ async def onboard_ecosystem_tenant(
 
 
 async def onboard_issuer(
-    *, name: str, issuer_controller: AcaPyClient, endorser_controller: AcaPyClient
+    *,
```

Comment:
I know this is not a change but looking at this rn I'm wondering why we have the * unpacking here?

Reply:
thanks for clarifying